src/lib/phdi-building-blocks/phdi_building_blocks/conversion.py

```diff
@@ -334,18 +337,12 @@ def convert_message_to_fhir(
         headers=headers,
         data=data,
     )
+    print(response)
```

Comment:
I think we might want to use the logging module here maybe at the info level. I have observed odd behavior with `print` in the logs of Azure environments.

Reply:
oops this is quick and dirty debug code that shouldn't have made it in - I'll remove it.

Reply:
done.